src/run/run.c

```diff
@@ -177,6 +181,7 @@ static int help(void) {
                "                                  when queueing a new job\n"
                "     --ignore-failure             Ignore the exit status of the invoked process\n"
                "     --background=COLOR           Set ANSI color for background\n"
+               "     --no-pager                   Do not pipe output into a pager\n"
```

Comment:
Hmm, in the specific case of systemd-run kinda tempted to mention this applies to --help only.